src/collections/posts/comfy-sabers.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Comfy Sabers
+section: articles
+category: event
+publish: 2024-04-24T19:00:00.000Z
+image: /static/uploads/posts/comfy-sabers-no-text.png
+showInPostListing: false
+---
+
+<div class="image">
```

Comment:
Good! Works. Not sure why you wrap it in <p> though, but I guess that may have been about the issue with the logo?